main.go

```diff
@@ -443,3 +452,20 @@ This means that when the API changes
 	log.Donef("Success")
 	log.Printf("The app (.ipa) was successfully uploaded to [iTunes Connect](https://itunesconnect.apple.com), you should see it in the *Prerelease* section on the app's iTunes Connect page!")
 }
+
+func copyToTmp(pth string) (string, error) {
+	tmpDir, err := pathutil.NormalizedOSTempDirPath("ipaOrPkg")
+	if err != nil {
+		return "", err
+	}
+	tmpPath := filepath.Join(tmpDir, "tmp"+filepath.Ext(pth))
+	cmd := command.New("cp", pth, tmpPath)
```

Comment:
instead of cp, i think arsync would be a better choice: https://github.com/bitrise-io/go-utils/blob/master/command/file.go#L12